README.md

```diff
@@ -52,10 +53,38 @@ This can be caused from any of the following: Wrong location, not enough disk sp
 The customer gets an email from the panel to setup their account (incl. password) if they didn't have an account before. Otherwise they should be able to use their existing credentials.
 
 ## My game requires multiple ports allocated.
-Currently, this isn't possible with this module but is planned.
+Configure the "**Additional Ports**" option in the module settings.
+It expects a valid JSON string consisting of the parameter name or NONE, and a number representing a port as a numeric offset from the first available allocation.
+E.g: If you enter `{1:"NONE", 2:"NONE", 4:"NONE"}` and the first available port happens to be 25565, you'll get as additional allocations: 
+* 25566 (First Port + 1)
+* 25567 (First Port +2)
+* 25569 (First Port +4)
+
+(If they're available)
+
+Note: I this option is set, it will override anything specified under "port_range" - Use one or the other, not both.
```

Comment:
It will not only override port_range but also every under deploy.
That should be dedicated IP, location and port_range